### ISSUE TITLE
dinglehopper builds wheel with version 0.0.0

### ISSUE THREAD

**Issue by mikegerber** [qurator-spk/setuptools_ocrd#10]
```
❯ python3 -m pip install --upgrade build && python3 -m build
[...]
Successfully built dinglehopper-0.9.1.tar.gz and dinglehopper-0.0.0-py3-none-any.whl
```

**Comment by mikegerber** [comment]
The source package does not contain ocrd-tool.json in the top level directory, this is likely the source of the problem.

**Comment by mikegerber** [comment]
Worked around the problem by adding a `MANIFEST.in` in dinglehopper:

```
❯ cat MANIFEST.in
include ocrd-tool.json
```

This builds a wheel with the correct version:
```
Successfully built dinglehopper-0.9.1.tar.gz and dinglehopper-0.9.1-py3-none-any.whl
```

Now we need a real fix: setuptools-ocrd must instruct setuptools to include the file.

**Comment by mikegerber** [comment]
Relevant documentation is somewhat difficult to discover, but this seems to be it:

https://setuptools.pypa.io/en/latest/userguide/extension.html#adding-support-for-revision-control-systems

**Comment by mikegerber** [comment]
First idea would be to:
a. call the original files_finder (How?)
b. Add ocrd-tool.json (both the symlink and the file it's pointing to)

**Comment by mikegerber** [comment]
This is fixed in 0.0.18 by adding a `setuptools.file_finders` entry-point that adds `ocrd-tool.json` to the sdist. `python -m build` - the official PyPA building tool - uses the sdist to build the wheel.

👀 @bertsky This should now be ready for testing. 

dinglehopper now built with the correct versions:
https://github.com/qurator-spk/dinglehopper/actions/runs/5835678499/job/15827784391
